webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -186,12 +191,20 @@ class ManageManifestModal extends Component {
                     {__('Red Hat CDN URL')}
                   </ControlLabel>
                   <Col sm={9}>
-                    <FormControl
-                      id="cdnUrl"
-                      type="text"
-                      value={organization.redhat_repository_url || ''}
-                      onChange={this.saveOrganization}
-                    />
+                    <InputGroup>
+                      <FormControl
+                        id="cdnUrl"
+                        type="text"
+                        value={this.state.redhat_repository_url || organization.redhat_repository_url || ''}
+                        onChange={this.updateRepositoryUrl}
+                      />
+
+                      <InputGroup.Button>
+                        <Button bsStyle="primary" id="cdn-update-btn" onClick={this.saveOrganization}>
+                          {organization.loading ? <span className="fa fa-spinner fa-spin fa-fw" /> : __('Update')}
```

Comment:
Could we use the [`<Spinner>`](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-Loading=true&knob-Size=xs&selectedKind=patternfly-react%2FWidgets%2FSpinner&selectedStory=Spinner&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) component here?  Perhaps with inline and xs size set?